how to make a bootable usb.md

```diff
@@ -0,0 +1,31 @@
+# dahilaos
+
+## how to make a bootable usb
+
+### windows
+
+**64 bit**
+
+1 first download the **linuxmint 20 iso [here](https://linuxmint.com/edition.php?id=281)**.
+
+2 then flash the linuxmint iso to your usb with **rufus [here](https://rufus.ie/)**.
+
+3 now download the latest **dahilaosbuild.zip [here](https://github.com/HexaOneOfficial/dahilaos/releases/)**. 
+
+4 then unpack the zip file and than overwrite the files on your linuxmint usb drive. 
+
+**32 bit**
+
+1 first download the **linuxmint 19.3 32bit iso [here](https://www.linuxmint.com/edition.php?id=273/)**.
+
+2 then flash the linuxmint iso to your usb with **rufus [here](https://rufus.ie/)**.
+
+3 now download the latest 32 bit **dahilaosbuild.zip [here](https://github.com/HexaOneOfficial/dahilaos/releases/)**. 
+
+4 then unpack the zip file and than overwrite the files on your linuxmint usb drive. 
+
+***
+![img](https://github.com/dahlia-os/Icons/blob/master/UI-Screenshots/Pangolin-2020-23_04.png)
+
+thats all! thanks to the dahila os team and linuxmint team for making this possible.
+**copyright github.com/dahila-os**
```

Comment:
thanks appreciated but there's no need for these lines in official documentation :)

Reply:
fixed all of your problems and some more typo's